python/HeaderService/hscalc.py

```diff
@@ -33,7 +33,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def get_date(timeStamp=None, scale='tai'):
+def get_date(timeStamp=None, format='unix', scale='utc'):
```

Comment:
Looks like you need to add format and scale parameters to the function documentation.

Reply:
Thanks, done now,